openwisp_radius/api/serializers.py

```diff
@@ -347,6 +349,22 @@ class Meta:
 class PasswordResetSerializer(BasePasswordResetSerializer):
     password_reset_form_class = PasswordResetForm
 
+    def send_mail(
+        self,
+        subject_template_name,
+        email_template_name,
+        context,
+        to_email,
+        html_email_template_name=None,
+    ):
+        subject = loader.render_to_string(subject_template_name, context)
+        # Email subject *must not* contain newlines
+        subject = ''.join(subject.splitlines())
+        body = loader.render_to_string(email_template_name, context)
+        if html_email_template_name is not None:
+            body_html = loader.render_to_string(html_email_template_name, context)
+        send_email(subject, body, body_html, [to_email], context)
```

Comment:
I would try to do this differently, I would define a template only for the body message and pass everything else as extra context.

```python
# here I would only put the HTML to render the message and nothing else
body_html = loader.render_to_string(html_email_template_name, context)
# this should
send_email(subject, body, body_html, [to_email], extra_context={
    'subject': '.......',
    'call_to_action_url': '....',
    'call_to_action_text': _('Reset password')
})
```

Can you try this?